NeuroID/NeuroIDClass/NeuroID.swift

```diff
@@ -148,6 +153,10 @@ public enum NeuroID {
         DataStore.insertEvent(screen: event.type, event: event)
     }
 
+    internal static func saveQueuedEventToLocalDataStore(_ event: NIDEvent) {
```

Comment:
When are we calling this? Only setUserId?

Reply:
@cometman - Currently yes, but we want to leave open the possibility to do this with other events (I.e. setScreenName)